.github/PULL_REQUEST_TEMPLATE/add_feature.md

```diff
@@ -0,0 +1,12 @@
+**Current situation**
```

Comment:
This is the same as the feature request. What about this: https://github.com/iza-institute-of-labor-economics/gettsim/blob/master/.github/pull_request_template.md?

Reply:
Copied it, except for the `changes.rst` because we don't have one and I will only start to create one when we reach version `0.1.x`